dtfs-central-api/src/v1/controllers/portal/gef-deal/create-gef-deal.controller.js

```diff
@@ -13,7 +13,13 @@ const createDeal = async (deal) => {
 };
 
 exports.createDealPost = async (req, res) => {
-  const createdDeal = await createDeal(req.body);
+  const deal = req?.body;
+
+  if (typeof deal?.dealType !== 'string') {
+    return res.status(400).send({ status: 400, message: 'Invalid deal payload' });
```

Comment:
Should we say "Invalid dealType" instead so the user knows what is wrong with the request body?

Reply:
Have updated. Thanks